server/src/test/java/org/elasticsearch/common/metrics/ExponentialBucketHistogramTests.java

```diff
@@ -73,7 +73,7 @@ public void testHistogramRandom() {
             expectedCounts[bucket] += 1;
 
             final int lowerBound = bucket == 0 ? 0 : upperBounds[bucket - 1];
-            final int upperBound = bucket == upperBounds.length ? randomBoolean() ? 100000 : Integer.MAX_VALUE : upperBounds[bucket] - 1;
+            final int upperBound = bucket == upperBounds.length ? Integer.MAX_VALUE : upperBounds[bucket] - 1;
```

Comment:
This was supposed to prioritise lower, more realistic numbers sometimes, but that seems unnecessary. It failed when I randomised the number of buckets (it was fixed at 18 previously)